CMakeLists.txt

```diff
@@ -15,7 +15,7 @@ include_directories(AFTER, vendor/spdlog/include)
 find_package(CURL REQUIRED)
 include_directories(${CURL_INCLUDE_DIRS})
 
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wall")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++17 -Wall")
```

Comment:
Bleeding edge, aren't we?

Reply:
Not really, but I suppose that GCC 8.x (which is the default in CentOS-like systems now) should be fine with this.